org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
+    "finish-args": ["--command=pitivi",
+                    "--share=ipc",
+                    "--socket=x11",
+                    "--socket=session-bus",
```

Comment:
`--socket=session-bus` gives you unrestricted access to the dbus session. You should be using `--talk-name=foo.bar` instead.